recipes/gravelamps/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "gravelamps" %}
+{% set version = "2.3.1rc14" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://git.ligo.org/mick.wright/Gravelamps/-/archive/{{ version }}/Gravelamps-{{ version }}.tar.gz
+  sha256:  95569b7a1bf3b44574b3915da1bd7404434d8c3823b8d7e62849eab8d4950bf5
+
+build:
+  entry_points:
+    - gravelamps_inference=gravelamps.inference:main
+    - gravelamps_generate_lens=gravelamps.generate_lens:main
+    - gravelamps_generate_interpolator_data=gravelamps.lensing.generic:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [not linux or py<=39]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make
+    - sysroot_linux-64
+  host:
+    - python
+    - setuptools-scm
+    - pip
+    - make
+    - arb
+    - libflint
+    - libgomp
+    - boost-cpp
+  run:
+    - python
+    - asimov
+    - astropy
+    - bilby
+    - bilby_pipe
+    - python-htcondor
+    - numpy
+    - scipy
+
+test:
+  imports:
+    - gravelamps
+    - gravelamps.lensing.point
+  commands:
+    - pip check
+    - gravelamps_inference --help
+    - gravelamps_generate_lens --help
+    - gravelamps_generate_interpolator_data --help
+  requires:
+    - pip
+    - sysroot_linux-64
+    - libstdcxx-ng
+    - arb
+    - libflint
```

Comment:
```suggestion
```
If you have to specify any of these packages in test requirements, it means your package dependencies are incomplete. Or the upstream packages are missing run_exports.